CIPs/cip-0064.md

```diff
@@ -0,0 +1,62 @@
+---
+cip: 64
+title: "New Transaction Type: Celo Dynamic Fee v2"
+author: Gastón Ponti (@gastonponti)
+discussions-to: https://github.com/celo-org/celo-proposals/discussions/393
+status: Accepted
+type: Standards Track
+category: Ring 0
+created: 2023-07-10
+license: Apache 2.0
+---
+
+## Simple Summary
+
+Create a new [EIP-2718](https://eips.ethereum.org/EIPS/eip-2718) Transaction Type that will act as [eip1559](https://eips.ethereum.org/EIPS/eip-1559) tx but including the `feeCurrency` field for paying in a different currency.
+
+## Abstract
+
+This CIP proposes to create a new EIP-2718 Transaction Type `0x7b`, which will have the same functionality as the [CIP-42](https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-0042.md), but removing the `gateway` fields (already deprecated in the [CIP-57](https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-0057.md)).
+With the format `0x7b || rlp([chainId, nonce, maxPriorityFeePerGas, maxFeePerGas, gasLimit, to, value, data, accessList, feeCurrency, signatureYParity, signatureR, signatureS])`
```

Comment:
This sentences misses some parts. Something like: *This transaction is encoded as follows.*